### DISCUSSION TITLE
Unable to override wrapper built-in component

### DISCUSSION THREAD

**Discussion by krishnaeverestengineering** [GrapesJS/grapesjs#5169] [Q&A]
editor.Components.addType('wrapper', {
 model: {
   defaults: {
     tagName: 'div', // use `div` instead of `body` in HTML export
   },

   // Skip wrapper in the HTML output
   toHTML(opts) {
     return this.getInnerHTML(opts);
   }
 }
})

I'm trying to override built-in wrapper but this doesn't work me. Wrapper is adding body tag which I don't need. 

How to fix it? Any help would be appreciate. Thanks

**Comment by artf** [comment]
https://github.com/GrapesJS/grapesjs/discussions/4499#discussioncomment-6162630
